geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/AccumuloParams.scala

```diff
@@ -25,7 +27,7 @@ trait AccumuloRasterTableParam {
   var table: String = null
 }
 
-trait AccumuloConnectionParams extends OptionalZookeepersParam {
```

Comment:
The only downside here is that now we are mixing the params classes with some logic for the environment...may not matter...I thought we separated it out earlier so we could use these standalone but not sure...let me think